welcome-to-sourcery.py

```diff
@@ -1,13 +1,76 @@
 
-# Welcome to Sourcery!
-
-# To get started log into your Sourcery account.
-# Open the command palette (Ctrl/Cmd+Shift+P) and execute `Sourcery: login`.
-
-# We're here to help you write cleaner, clearer code.
-# When you log in to Sourcery you'll start to see parts 
-# of your code like this start to get underlined. 
-# This means Sourcery has a suggestion.
+# Welcome to Sourcery! We're here to be your pair programmer
+# anytime you're working in VS Code.
+
+# To get started log into your Sourcery account. Click on
+# the Sourcery logo (the hexagon) on your VS Code sidebar
+# and click the button to log in.
+
+# Or, open the command palette (Ctrl/Cmd+Shift+P) and
+# execute `Sourcery: login`.
+
+# Sourcery works in 2 ways:
+# 1. Gives you instant suggestions for improvements and
+# refactorings to your Python, JavaScript, and TypeScript
+# code. All of this runs fully locally
+# 2. Acts as a pair programmer allowing you to ask it
+# questions, write new code, and interact with existing code.
+
+# To start using the pair programmer seection of Sourcery,
+# click the Sourcery sidebar option and clic the Opt In button.
+
+# Now you can start asking Sourcery questions or asking it
+# to interact with sections of your code. Let's take a look
+# at a few examples:
+
+# Above each function you'll see a few commands - these are
+# Code Lenses that you can use to interact with Sourcery. 
+# Try clicking the "Ask Sourcery" Code Lens and asking it to
+# update the code to use `dateutil`
+
+
+def days_between_dates(date1, date2):
+    d1 = datetime.datetime.strptime(date1, '%Y-%m-%d').date()
+    d2 = datetime.datetime.strptime(date2, '%Y-%m-%d').date()
+    delta = d2 - d1
+    return delta.days
+
+# With the Ask Sourcery command or the chat in the sidebar you
+# can ask Sourcery questions, have it write new code for you, or
+# update existing code.
+
+# Sourcery has a series of built in "recipes" you can quickly use
+# to interact with sections of code. 
+
+# Try clicking the Explain Code lens above this next function:
```

Comment:
I would think about starting with Generate Docstrings as it might be more immediately appealing to more people?